index.src.html

```diff
@@ -459,6 +465,119 @@
+        without user involvement, wonderful. If user involvement is necessary, then the operation
+        will return `null` rather than involving the user.
+
+        Note: The intended usage is to support ["Keep me signed-into this site"
+        scenarios](#example-mediation-silent), where a developer may wish to silently obtain
+        credentials if a user should be automatically signed in, but to delay bothering the user
+        with a sign-in prompt until they actively choose to sign-in.
+
+    :   <dfn>optional</dfn>
+    ::  If credentials can be handed over for a given operation without user mediation, they will
+        be. If [=requires user mediation|user mediation is required=], then the user agent will
+        involve the user in the decision.
+
+        Note: This is the default behavior for {{get()}}, and is intended to serve a case where a
+        developer has reasonable confidence that a user expects to start a sign-in operation. If
+        a user has just clicked "sign-in" for example, then they won't be surprised or confused to
+        see a [=credential chooser=] if necessary.
+
+    :   <dfn>required</dfn>
+    ::  The user agent will not hand over credentials without [=user mediation=], even if the
+        [=requires user mediation=] flag is unset for an origin.
+
+        Note: This requirement is intended to support [reauthentication](#example-mediation-require)
+        or [user-switching](#example-mediation-switch) scenarios. Further, the requirement is tied
+        to a specific operation, and does not affect the [=requires user mediation=] flag for the
+        origin. To toggle that flag, developers should call {{requireUserMediation()}}.
+  </dl>
+
+  #### Examples #### {#mediation-examples}
+
+  <div class="example" id="example-mediation-silent">
+    MegaCorp, Inc. wishes to seamlessly sign in users when possible. They can do so by calling
+    {{get()}} for all non-signed in users at some convinient point while a landing page is loading,
+    passing in a {{CredentialRequestOptions/mediation}} member set to
+    "{{CredentialMediationRequirement/silent}}". This ensures that users who have opted-into
+    dropping the requirements for user mediation (as described in [[#user-mediation-requirement]])
+    are signed in, and users who haven't opted-into such behavior won't be bothered with a confusing
+    [=credential chooser=] popping up without context:
+
+    <pre>
+      window.addEventListener('load', _ =&gt; {
+        var c = await navigatior.<a for="Navigator" attribute>credentials</a>.<a for="CredentialsContainer" method lt="get()">get</a>({
+          ...,
+          <a for="CredentialRequestOptions" dict-member>mediation</a>: "<a for="CredentialMediationRequirement" enum-value>silent</a>"
+        });
+        if (c) {
+          // Hooray! Let's sign the user in using these credentials!
+        }
+      });
+    </pre>
+  </div>
+
```

Comment:
Should we another example here "example-mediation-optional" for the sake of completeness?

MegaCorp, Inc. does not want to use silent sign-in but wishes to make the sign-in button on their homepage as convenient as possible. If exactly one credential is available and not mediation is required, that credential should be returned silently. Otherwise, the user can be prompted to pick a credential. --> optional?